### ISSUE TITLE
No longer removing DRM

### ISSUE THREAD

**Issue by Raeludwig** [noDRM/DeDRM_tools#433]
### Question / bug report

I normally use this without any issues, but recently it isn't removing the DRM anymore. I hadn't changed any settings or updated anything. Since the issue started I have updated Calibre and re-installed the plugin (I downloaded and tried to install v10.0.9 but when I install that it still says v10.0.3). Any help would be greatly appreciated!

### Which version of Calibre are you running?

7.0.0

### Which version of the DeDRM plugin are you running?

v10.0.3

### Log output

```log
calibre_plugins.dedrm.DeDRMError: DeDRM v10.0.3: Ultimately failed to decrypt after 0.0 seconds.
```

**Comment by ElleKayEm** [comment]
If it still says 10.0.3 after loading the plugin and restarting calibre, then you reinstalled the older version.  If the book is in kfx format, you also need the KFX Input plugin.

**Comment by bobbydriver** [comment]
Same here - updated to Calibre 7 today and DRM no longer being removed. Updated DeDRM to 10.0.9 and still not working

**Comment by ElleKayEm** [comment]
Post a log and we'll take a look.

**Comment by bobbydriver** [comment]
Hmm - looks like it's a B&N epub that someone shared with me. I don't have a Nook, so I guess I'm not reading it

**Comment by ElleKayEm** [comment]
Yeah, you're not going to be able to remove DRM that's not tied to your own account.
